[2024-05-04T11:35:01.418+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-04T11:35:01.702+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.upload_file_sellers_to_capstone scheduled__2024-05-03T00:00:00+00:00 [queued]>
[2024-05-04T11:35:01.783+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.upload_file_sellers_to_capstone scheduled__2024-05-03T00:00:00+00:00 [queued]>
[2024-05-04T11:35:01.787+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-05-04T11:35:01.952+0000] {taskinstance.py:2327} INFO - Executing <Task(GCSToGCSOperator): upload_file_sellers_to_capstone> on 2024-05-03 00:00:00+00:00
[2024-05-04T11:35:02.150+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=9141) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-04T11:35:02.152+0000] {standard_task_runner.py:63} INFO - Started process 9248 to run task
[2024-05-04T11:35:02.177+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'upload_file_sellers_to_capstone', 'scheduled__2024-05-03T00:00:00+00:00', '--job-id', '165', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpwap0osff']
[2024-05-04T11:35:02.214+0000] {standard_task_runner.py:91} INFO - Job 165: Subtask upload_file_sellers_to_capstone
[2024-05-04T11:35:02.409+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.upload_file_sellers_to_capstone scheduled__2024-05-03T00:00:00+00:00 [running]> on host d4c1d9b975b8
[2024-05-04T11:35:04.601+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='upload_file_sellers_to_capstone' AIRFLOW_CTX_EXECUTION_DATE='2024-05-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-03T00:00:00+00:00'
[2024-05-04T11:35:04.604+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-04T11:35:04.672+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-05-04T11:35:04.697+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-05-04T11:35:05.728+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-04T11:35:05.776+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=upload_file_sellers_to_capstone, execution_date=20240503T000000, start_date=20240504T113501, end_date=20240504T113505
[2024-05-04T11:35:06.035+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-04T11:35:06.760+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-04T11:35:06.786+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
